gnat2goto/examples/if_expr.adb

```diff
@@ -0,0 +1,5 @@
+procedure If_Expr (B : Boolean) is
```

Comment:
We should add these things to the "unit" test directory (even though they're not unit tests...)